lib/srv/discovery/fetchers/eks.go

```diff
@@ -360,7 +364,7 @@ func (a *eksFetcher) getMatchingKubeCluster(ctx context.Context, clusterName str
 
 	switch st := aws.StringValue(rsp.Cluster.Status); st {
 	case eks.ClusterStatusUpdating, eks.ClusterStatusActive:
-		a.Log.WithField("cluster_name", clusterName).Debugf("EKS cluster status is valid: %s", st)
+		a.Logger.DebugContext(ctx, "EKS cluster status is valid", "status", st, "cluster_name", clusterName)
```

Comment:
```suggestion
		a.Logger.DebugContext(ctx, "EKS cluster status is valid", "status", st, "cluster", clusterName)
```

To match other calls in this file.